lib/swaggerValidator/ajvSchemaValidator.ts

```diff
@@ -179,7 +183,7 @@ export const ajvErrorToSchemaValidateIssue = (
     message: errInfo.message,
     params: errInfo.params,
     jsonPathsInPayload: [dataPath],
-    schemaPath: err.schemaPath,
+    schemaPath: errInfo.code === "MISSING_RESOURCE_ID" ? "" : err.schemaPath,
```

Comment:
Why do we need to hard code this logic?

Reply:
Doesn't want to set it as '#/x-ms-azureresource' and leave it as ''.